src/thehexbot/Program.cs

```diff
@@ -8,7 +8,7 @@
 using Tweetinvi;
 using Tweetinvi.Parameters;
 using Tweetinvi.Models;
-
+using config = thehexbot.models;
```

Comment:
why this like? A simple `using thehexbot.Models` should allow us to use `Configuration` right? Why `config.Configuration`? It seems redundant